### ISSUE TITLE
Remove unused GetReviewers

### ISSUE THREAD

**Issue by meling** [quickfeed/quickfeed#928]
The frontend no longer uses this method:
```typescript
public getReviewers(submissionID: bigint, courseID: bigint): Promise<IGrpcResponse<Reviewers>> {
```
and by extension, this RPC call is no longer needed:
```proto
rpc GetReviewers(SubmissionRequest) returns (Reviewers) {}
```
Fixing this should be done after #923 has been merged, or we can build on top of it.
